pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7656922?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7656922?buildTab=artifacts#/) on release-22.2 @ [f660a454f6781a379fcc2a17646dc859ac5af997](https://github.com/cockroachdb/cockroach/commits/f660a454f6781a379fcc2a17646dc859ac5af997):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2747705489
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2747705489
--- FAIL: TestTenantStatusAPI (3.83s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/974/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.01s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.21s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-observability @cockroachdb/multi-tenant
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7784570?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7784570?buildTab=artifacts#/) on release-22.2 @ [ca673993473629a46370100108dd8c185a592e91](https://github.com/cockroachdb/cockroach/commits/ca673993473629a46370100108dd8c185a592e91):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI184133135
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI184133135
--- FAIL: TestTenantStatusAPI (3.88s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2655/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7854033?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7854033?buildTab=artifacts#/) on release-22.2 @ [adf24a585553397a7ad8e4bb6da0609bb5d2e930](https://github.com/cockroachdb/cockroach/commits/adf24a585553397a7ad8e4bb6da0609bb5d2e930):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1131233108
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1131233108
--- FAIL: TestTenantStatusAPI (4.01s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2819/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.31s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910572?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910572?buildTab=artifacts#/) on release-22.2 @ [97d5489387634dfd25462c01cace49445d5698de](https://github.com/cockroachdb/cockroach/commits/97d5489387634dfd25462c01cace49445d5698de):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3267991451
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3267991451
--- FAIL: TestTenantStatusAPI (4.09s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2212/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.04s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7924615?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7924615?buildTab=artifacts#/) on release-22.2 @ [ff69f981a02ea6906d4be138189c4531a6a420d1](https://github.com/cockroachdb/cockroach/commits/ff69f981a02ea6906d4be138189c4531a6a420d1):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI821492505
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI821492505
--- FAIL: TestTenantStatusAPI (3.81s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2433/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8080616?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8080616?buildTab=artifacts#/) on release-22.2 @ [8ff653b220128d32bbd7eff2717d6212b2b6ac12](https://github.com/cockroachdb/cockroach/commits/8ff653b220128d32bbd7eff2717d6212b2b6ac12):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1227847817
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1227847817
--- FAIL: TestTenantStatusAPI (4.18s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2756/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.57s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8154746?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8154746?buildTab=artifacts#/) on release-22.2 @ [07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1](https://github.com/cockroachdb/cockroach/commits/07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1432284473
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1432284473
--- FAIL: TestTenantStatusAPI (3.77s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3039/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.06s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8241542?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8241542?buildTab=artifacts#/) on release-22.2 @ [cce52c9eab1210c6cd508f6b99b2bad0ba3e1e56](https://github.com/cockroachdb/cockroach/commits/cce52c9eab1210c6cd508f6b99b2bad0ba3e1e56):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1590134114
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1590134114
--- FAIL: TestTenantStatusAPI (4.19s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2290/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.11s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8396532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8396532?buildTab=artifacts#/) on release-22.2 @ [cce0718a324e36d4144cc44a61186d26043b5601](https://github.com/cockroachdb/cockroach/commits/cce0718a324e36d4144cc44a61186d26043b5601):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1500189559
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1500189559
--- FAIL: TestTenantStatusAPI (3.74s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3579/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.05s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95341 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-sql-observability branch-release-22.2.2-with-dms-changes]
- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8441640?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8441640?buildTab=artifacts#/) on release-22.2 @ [82ac4cd732996b6e42870ce7e18306ecab214308](https://github.com/cockroachdb/cockroach/commits/82ac4cd732996b6e42870ce7e18306ecab214308):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4263429121
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4263429121
--- FAIL: TestTenantStatusAPI (4.22s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.31s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2905/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95341 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-sql-observability branch-release-22.2.2-with-dms-changes]
- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8494947?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8494947?buildTab=artifacts#/) on release-22.2 @ [799d8dbba62fa9c942a780584c71f8ceab784381](https://github.com/cockroachdb/cockroach/commits/799d8dbba62fa9c942a780584c71f8ceab784381):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1454185378
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1454185378
--- FAIL: TestTenantStatusAPI (4.17s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1439/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.31s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95341 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2.2-with-dms-changes]
- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8552254?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8552254?buildTab=artifacts#/) on release-22.2 @ [92959b31f134f9d8a0a60c51276c82635a18c2a8](https://github.com/cockroachdb/cockroach/commits/92959b31f134f9d8a0a60c51276c82635a18c2a8):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3103746049
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3103746049
--- FAIL: TestTenantStatusAPI (4.43s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3538/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8588503?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8588503?buildTab=artifacts#/) on release-22.2 @ [9efe3c5c73d3a5eb54b3c4bfe2e3a83504702c39](https://github.com/cockroachdb/cockroach/commits/9efe3c5c73d3a5eb54b3c4bfe2e3a83504702c39):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI414070933
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI414070933
--- FAIL: TestTenantStatusAPI (4.14s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2983/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.12s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8612336?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8612336?buildTab=artifacts#/) on release-22.2 @ [7cdf538ae6f0ff249d397ab64dbf144a543c5d1b](https://github.com/cockroachdb/cockroach/commits/7cdf538ae6f0ff249d397ab64dbf144a543c5d1b):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI448661187
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:113: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI448661187
--- FAIL: TestTenantStatusAPI (3.87s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1179: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1649/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1179
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.17s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8669499?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8669499?buildTab=artifacts#/) on release-22.2 @ [beb24ea0688dcb96ca8a1f2ee85cd154efe0c0b2](https://github.com/cockroachdb/cockroach/commits/beb24ea0688dcb96ca8a1f2ee85cd154efe0c0b2):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI659986206
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI659986206
--- FAIL: TestTenantStatusAPI (3.92s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3503/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8682560?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8682560?buildTab=artifacts#/) on release-22.2 @ [6ab0496d81d303bae3854ee810fe2233ce18791f](https://github.com/cockroachdb/cockroach/commits/6ab0496d81d303bae3854ee810fe2233ce18791f):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2635088526
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2635088526
--- FAIL: TestTenantStatusAPI (4.10s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3375/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.05s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726868?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726868?buildTab=artifacts#/) on release-22.2 @ [6bb5cbc789500d4336e46042ae5ba32b2c466514](https://github.com/cockroachdb/cockroach/commits/6bb5cbc789500d4336e46042ae5ba32b2c466514):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI50091688
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI50091688
--- FAIL: TestTenantStatusAPI (4.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/10/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.57s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8751485?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8751485?buildTab=artifacts#/) on release-22.2 @ [31eb3ce6ceb18123fd2e7eac464b25e203d7de0f](https://github.com/cockroachdb/cockroach/commits/31eb3ce6ceb18123fd2e7eac464b25e203d7de0f):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI47394738
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI47394738
--- FAIL: TestTenantStatusAPI (5.15s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2328/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8773405?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8773405?buildTab=artifacts#/) on release-22.2 @ [31eb3ce6ceb18123fd2e7eac464b25e203d7de0f](https://github.com/cockroachdb/cockroach/commits/31eb3ce6ceb18123fd2e7eac464b25e203d7de0f):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1470586518
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1470586518
--- FAIL: TestTenantStatusAPI (3.87s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1083/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.01s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8821102?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8821102?buildTab=artifacts#/) on release-22.2 @ [ff72f9a792e247c920b6b75310309f967017f8b1](https://github.com/cockroachdb/cockroach/commits/ff72f9a792e247c920b6b75310309f967017f8b1):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3695165750
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
--- FAIL: TestTenantStatusAPI (4.09s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3873/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.31s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8844726?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8844726?buildTab=artifacts#/) on release-22.2 @ [c9b71bafa2857ebdc5ffea629fb9546937f10ba9](https://github.com/cockroachdb/cockroach/commits/c9b71bafa2857ebdc5ffea629fb9546937f10ba9):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2503368898
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2503368898
--- FAIL: TestTenantStatusAPI (3.90s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2963/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.09s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8851835?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8851835?buildTab=artifacts#/) on release-22.2 @ [c9b71bafa2857ebdc5ffea629fb9546937f10ba9](https://github.com/cockroachdb/cockroach/commits/c9b71bafa2857ebdc5ffea629fb9546937f10ba9):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1938886929
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1938886929
--- FAIL: TestTenantStatusAPI (3.86s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1207: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2908/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1207
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.07s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92979 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-master skipped-test]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10027398?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10027398?buildTab=artifacts#/) on release-22.2 @ [163c2296507466cb3fb833bfd32484a62b405460](https://github.com/cockroachdb/cockroach/commits/163c2296507466cb3fb833bfd32484a62b405460):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1433419364
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1433419364
--- FAIL: TestTenantStatusAPI (4.20s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1185: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2975/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1185
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102224 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1.0]
- #99770 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1]
- #99559 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10202053?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10202053?buildTab=artifacts#/) on release-22.2 @ [62a0f7dd692b164236a0a540c9f4ed70b3db817b](https://github.com/cockroachdb/cockroach/commits/62a0f7dd692b164236a0a540c9f4ed70b3db817b):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2518498319
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2518498319
--- FAIL: TestTenantStatusAPI (3.98s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1185: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2991/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1185
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.17s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102224 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability todo-deprecate.branch-release-23.1.0]
- #99770 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1]
- #99559 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10291535?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10291535?buildTab=artifacts#/) on release-22.2 @ [afe2d46600257f6c022ce89168bebdc0023c8215](https://github.com/cockroachdb/cockroach/commits/afe2d46600257f6c022ce89168bebdc0023c8215):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI656253029
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI656253029
--- FAIL: TestTenantStatusAPI (4.03s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1185: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3333/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1185
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.20s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102224 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability todo-deprecate.branch-release-23.1.0]
- #99770 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1]
- #99559 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10485621?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10485621?buildTab=artifacts#/) on release-22.2 @ [8a2050b00bac7e247df26c63aa0c8c1ca4fd7996](https://github.com/cockroachdb/cockroach/commits/8a2050b00bac7e247df26c63aa0c8c1ca4fd7996):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3204178741
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3204178741
--- FAIL: TestTenantStatusAPI (3.85s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1185: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2200/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1185
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.05s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102224 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability todo-deprecate.branch-release-23.1.0]
- #99770 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1]
- #99559 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10579851?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10579851?buildTab=artifacts#/) on release-22.2 @ [8055184fc991424d1d0eefd7ccf703948b8de3ee](https://github.com/cockroachdb/cockroach/commits/8055184fc991424d1d0eefd7ccf703948b8de3ee):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3005428874
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3005428874
--- FAIL: TestTenantStatusAPI (4.25s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1185: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/statusccl_test/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1185
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.31s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102224 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability todo-deprecate.branch-release-23.1.0]
- #99770 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1]
- #99559 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10596048?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10596048?buildTab=artifacts#/) on release-22.2 @ [15505faf5a63b374b40b2d8b02e035e5f2e31180](https://github.com/cockroachdb/cockroach/commits/15505faf5a63b374b40b2d8b02e035e5f2e31180):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI850833128
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI850833128
--- FAIL: TestTenantStatusAPI (3.78s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1185: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/statusccl_test/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1185
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.08s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102224 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability todo-deprecate.branch-release-23.1.0]
- #99770 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-release-23.1]
- #99559 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-cluster-observability branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/serverccl/statusccl/tenant_status_test.go
+++ b/pkg/ccl/serverccl/statusccl/tenant_status_test.go
@@ -1169,7 +1169,6 @@ func testTenantRangesRPC(_ context.Context, t *testing.T, helper serverccl.Tenan
 	})
 
 	t.Run("test tenant ranges pagination", func(t *testing.T) {
-		skip.UnderStressWithIssue(t, 92382)
 		ctx := context.Background()
 		resp1, err := tenantA.TenantRanges(ctx, &serverpb.TenantRangesRequest{
 			Limit: 1,
@@ -1180,6 +1179,28 @@ func testTenantRangesRPC(_ context.Context, t *testing.T, helper serverccl.Tenan
 			require.Len(t, ranges.Ranges, 1)
 		}
 
+		// Note: This is a 22.2 compatible work around for the
+		// flake fix introduced in #99054.
+		// In 22.2, crdb_internal.ranges is not yet compatible with
+		// secondary tenants, so instead we rely on TenantRanges
+		// itself with the `limit` param omitted. The method is the same:
+		// We count the number of ranges belonging to the tenant and only proceed
+		// when we know there are enough ranges for subsequent calls
+		// with pagination to succeed.
+		testutils.SucceedsSoon(t, func() error {
+			res, err := tenantA.TenantRanges(ctx, &serverpb.TenantRangesRequest{})
+			require.NoError(t, err)
+			require.Equal(t, 1, len(res.RangesByLocality))
+			for k := range res.RangesByLocality {
+				rangeCount := len(res.RangesByLocality[k].Ranges)
+				if rangeCount < 3 {
+					return errors.Newf("expected >= 3 ranges, got %d", rangeCount)
+				}
+			}
+			return nil
+		})
+
+		// We can proceed now that we know there are at least 3 ranges.
 		resp2, err := tenantA.TenantRanges(ctx, &serverpb.TenantRangesRequest{
 			Limit:  1,
 			Offset: resp1.Next,

